---a/net/minecraft/network/play/server/S29PacketSoundEffect.java
+++b/net/minecraft/network/play/server/S29PacketSoundEffect.java
@@ -1,15 +1,16 @@
 package net.minecraft.network.play.server;
 
 import java.io.IOException;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.util.MathHelper;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
+
 import org.apache.commons.lang3.Validate;
 
-public class S29PacketSoundEffect implements Packet
-{
+public class S29PacketSoundEffect implements Packet {
+
     private String field_149219_a;
     private int field_149217_b;
     private int field_149218_c = Integer.MAX_VALUE;
@@ -20,21 +20,22 @@
     private int field_149214_f;
     private static final String __OBFID = "CL_00001309";
 
-    public S29PacketSoundEffect() {}
+    public S29PacketSoundEffect() {
+    }
 
-    public S29PacketSoundEffect(String p_i45200_1_, double p_i45200_2_, double p_i45200_4_, double p_i45200_6_, float p_i45200_8_, float p_i45200_9_)
-    {
+    public S29PacketSoundEffect(String p_i45200_1_, double p_i45200_2_, double p_i45200_4_, double p_i45200_6_, float p_i45200_8_, float p_i45200_9_) {
         Validate.notNull(p_i45200_1_, "name", new Object[0]);
         this.field_149219_a = p_i45200_1_;
-        this.field_149217_b = (int)(p_i45200_2_ * 8.0D);
-        this.field_149218_c = (int)(p_i45200_4_ * 8.0D);
-        this.field_149215_d = (int)(p_i45200_6_ * 8.0D);
+        this.field_149217_b = (int) (p_i45200_2_ * 8.0D);
+        this.field_149218_c = (int) (p_i45200_4_ * 8.0D);
+        this.field_149215_d = (int) (p_i45200_6_ * 8.0D);
         this.field_149216_e = p_i45200_8_;
-        this.field_149214_f = (int)(p_i45200_9_ * 63.0F);
+        this.field_149214_f = (int) (p_i45200_9_ * 63.0F);
         p_i45200_9_ = MathHelper.clamp_float(p_i45200_9_, 0.0F, 255.0F);
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.field_149219_a = buf.readStringFromBuffer(256);
         this.field_149217_b = buf.readInt();
         this.field_149218_c = buf.readInt();
@@ -44,8 +44,7 @@
         this.field_149214_f = buf.readUnsignedByte();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeString(this.field_149219_a);
         buf.writeInt(this.field_149217_b);
         buf.writeInt(this.field_149218_c);
@@ -54,26 +54,11 @@
         buf.writeByte(this.field_149214_f);
     }
 
-    @SideOnly(Side.CLIENT)
-    public String func_149212_c()
-    {
-        return this.field_149219_a;
-    }
-
-    public void processPacket(INetHandlerPlayClient handler)
-    {
+    public void processPacket(INetHandlerPlayClient handler) {
         handler.handleSoundEffect(this);
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerPlayClient)handler);
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerPlayClient) handler);
     }
-
-    @SideOnly(Side.CLIENT)
-    public double func_149207_d()
-    {
-        return (double)((float)this.field_149217_b / 8.0F);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public double func_149211_e()
-    {
-        return (double)((float)this.field_149218_c / 8.0F);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public double func_149210_f()
-    {
-        return (double)((float)this.field_149215_d / 8.0F);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public float func_149208_g()
-    {
-        return this.field_149216_e;
-    }
-
-    @SideOnly(Side.CLIENT)
-    public float func_149209_h()
-    {
-        return (float)this.field_149214_f / 63.0F;
-    }
-}
+}
